### PR TITLE
Allow plugin bundle testing

### PR DESCRIPTION
**Pull request by i11**
Allow plugin testing with bundle.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,4 +1,9 @@
 source 'https://rubygems.org'
 
-# Specify your gem's dependencies in vagrant-ovirt.gemspec
-gemspec
+group :development do
+  gem 'vagrant', :git => 'git://github.com/mitchellh/vagrant.git'
```
Comment (myoung34) [inline Gemfile]: vagrant is no longer supported as a gem, and their latest gem is 1.5.0 as compared to the 1.7.2 release

File: Gemfile [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
+end
+
+group :plugins do
+  gem 'vagrant-ovirt3', :path => '.'
```
Comment (myoung34) [inline Gemfile]: How does this work compared to gemspec? Will it still load rbovirt/fog?

File: Gemfile (hunk 1 above)
Comment (i11) [inline Gemfile]: Gem is taken from the source defined by :git key

File: Gemfile (hunk 2 above)
Comment (i11) [inline Gemfile]: As far as I understand gemspec defines a :default group for gems with no explicit group and :development group for development dependencies.

The change though switches the scope to vagrant gem source and adds the current source tree under plugins group (with all dependencies) which makes it possible to use it as a vagrant plugin through bundler.

File: Gemfile (hunk 1 above)
Comment (myoung34) [inline Gemfile]: oops. 8am me tl;dr'ed

**Comment by myoung34** [comment]
I'll verify this change as much as I can. As long as it builds the .gem fine with those dependencies, then I'm OK with this change and will merge it.  On a side note, the other 2 PR's you had went into a 1.1.1 that's been pushed to rubygems.org

**Comment by i11** [comment]
Sure thing.
Nice! Thank you for being so quick!

**Pull request merged**